src/einsteinpy/tests/test_coordinates/test_coord_transform.py

```diff
@@ -104,6 +104,32 @@ def test_cycle_BLSpherical(bl):
 # Tests for object.__repr__ and object.__str__
 
 
+def test_norm():
```

Comment:
slight change in function name
```suggestion
def test_cartesian_norm():
```